checklists/editor-checklist-peer-reviewed-pkg.csv

```diff
@@ -1,4 +1,4 @@
 YAML package-version included
-YAML subject tags - software peer review, packagename, R; add "community" for post by non-staff non-editor
+YAML subject tags - software peer review, packagename, R
```

Comment:
`R` tag no longer needed. Please remove from checklists

Reply:
You mean I should remove the "R" tag, right? Not community...

Reply:
Ah, and remove the community because that belongs in the other checklist, but also remove R :D